python/ray/ray_operator/main.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+from ray.ray_operator.operator_utils import NAMESPACED_OPERATOR
+from ray.ray_operator.operator_utils import OPERATOR_NAMESPACE
+
+
+def main() -> None:
+    args = ["--namespace", OPERATOR_NAMESPACE] if NAMESPACED_OPERATOR else [
+        "--all-namespaces"
+    ]
+    subprocess.run(
+        ["kopf", "run", *args, "-m", "ray.ray_operator.operator"],
```

Comment:
nit: seems kopf can run in a thread instead of a process
https://kopf.readthedocs.io/en/stable/embedding/